lightning/mydump/parquet_parser.go

```diff
@@ -89,7 +90,7 @@ func NewParquetParser(
 	for i, c := range reader.SchemaHandler.SchemaElements {
 		if c.GetNumChildren() == 0 {
 			// the SchemaElement.Name is capitalized, we should use the original name
-			columns = append(columns, reader.SchemaHandler.Infos[i].ExName)
+			columns = append(columns, strings.ToLower(reader.SchemaHandler.Infos[i].ExName))
```

Comment:
is the comment "the SchemaElement.Name is capitalized, we should use the original name" still relevant since we're always using `ToLower` here

Reply:
yes, in this way there is no difference between these two fields.

Reply:
i mean if they are the same we could simplify the loop to 

```go
for _, c := range reader.SchemaHandler.SchemaElements {
    if c.GetNumChildren() == 0 {
        columns = append(columns, strings.ToLower(c.Name))
```